src/sentry/migrations/0112_backfill_snuba_query_event_types.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-10-13 22:22
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from sentry.utils.query import RangeQuerySetWrapper
+
+
+def backfill_snuba_query_event_type(apps, schema_editor):
+    """
+    This backfills all SnubaQuery rows that don't have a `SnubaQueryEventType`. There's
+    just a
+    """
+    SnubaQuery = apps.get_model("sentry", "SnubaQuery")
+    SnubaQueryEventType = apps.get_model("sentry", "SnubaQueryEventType")
+
+    for snuba_query in RangeQuerySetWrapper(SnubaQuery.objects.all()):
+        if not SnubaQueryEventType.objects.filter(snuba_query=snuba_query).exists():
+            # 0 is SnubaQueryEventType.EventTypes.ERROR,
+            # 2 is SnubaQueryEventType.EventTypes.TRANSACTION.
+            SnubaQueryEventType.objects.create(
+                snuba_query=snuba_query, type=(0 if snuba_query.dataset == "events" else 2)
+            )
```

Comment:
I ran this locally with both event and transaction queries and it successfully backfilled the rows.